crates/bevy_transform/src/components/global_transform.rs

```diff
@@ -9,33 +9,62 @@ use bevy_reflect::{std_traits::ReflectDefault, Reflect};
 #[cfg(all(feature = "bevy-support", feature = "serialize"))]
 use bevy_reflect::{ReflectDeserialize, ReflectSerialize};
 
-/// [`GlobalTransform`] is an affine transformation from entity-local coordinates to worldspace coordinates.
+/// Describes the absolute transform (translation, rotation, scale) of an entity in space,
```

Comment:
The `(translation, rotation, scale)` part is a bit misleading, since `GlobalTransform` is not a TRS transformation in general. This also applies to the later part of this doc comment which breaks down translation, rotation, and scale.

Reply:
Right, I might see if I can walk that part back a bit.

Reply:
Would you mind re-checking, just to see if I've understood you correctly? Cheers!